static_src/components/app_list.jsx

```diff
@@ -87,26 +76,36 @@ export default class AppList extends React.Component {
         <table sortable>
           <thead>
             <tr>
-            { this.columns.map((column) =>
-              <th column={ column.label } className={ column.key }
-                key={ column.key }>
-                { column.label }
+              <th>
+                Apps in <EntityIcon entity="space" />{ this.state.currentSpaceName }
+              </th>
+              <th>
+                Memory used
+              </th>
+              <th>
+                Memory allocated
               </th>
-            )}
             </tr>
           </thead>
           <tbody>
           { this.state.apps.map((app) => {
+            let crashed = (app.state === appStates.crashed) &&
+              (<span className={ this.styler('status', app.state.toLowerCase()) }>
+                { app.state.toLowerCase() }
+              </span>);
             return ([
               <tr key={ app.guid }>
                 <td label="Name">
-                  <a href={ this.appUrl(app) }>{ app.name }</a>
+                  <h4 className={ this.styler('sans-s5') }>
+                    <EntityIcon entity="app" state={ app.state } />
+                    <span>
+                      { this.state.currentSpaceName } / { this.appName(app) }
+                    </span>
+                    <span>{ crashed }</span>
+                  </h4>
                 </td>
-                <td label="Buildpack">{ app.buildpack }</td>
-                <td label="Memory">{ app.memory } MB</td>
-                <td label="Instances">{ app.instances }</td>
-                <td label="State">{ app.state }</td>
-                <td label="Disk quota">{ app.disk_quota } MB</td>
+                <td label="Usage">{ app.memory } MB</td>
```

Comment:
@thisisdano I think we want to be careful with language here. The number here is really the memory allocated to the app, not the amount of memory being used.